src/main/cpp/subsystems/SwerveDrive.cpp

```diff
@@ -140,27 +140,38 @@ bool SwerveDrive::SetIdleMode(rev::CANSparkMax::IdleMode mode) {
 }
 
 bool SwerveDrive::VectorTeleOpDrive(const double &drive, const double &strafe, const double &turn) {
-    double final_drive = drive * control_decay + last_drive * (1 - control_decay);
-    double final_strafe = strafe * control_decay + last_strafe * (1 - control_decay);
-    double final_turn = turn * control_decay  + last_turn * (1 - control_decay);
+    double drive_ = drive;
+    double strafe_ = strafe;
+    double turn_ = turn;
 
     // if turn is very small
-    if (abs(final_turn) < 0.3) {
-        final_turn = 0.0; // then zero it
+    if (abs(turn) < 0.3) {
+        turn_ = 0.0; // then zero it
     // } else if (final_turn)
     }
 
     // if strafe is very small
-    if (abs(final_strafe) < 0.15) {
-        final_strafe = 0.0; // then zero it
+    if (abs(strafe) < 0.15) {
+        strafe_ = 0.0; // then zero it
     }
```

Comment:
Why don't we have a similar deadband for drive power?

Reply:
Consider this link as well for an "outer deadband" (also linked in that swerve doc): http://blog.hypersect.com/interpreting-analog-sticks/